### PR TITLE
{Release} Upgrade to Azure CLI 2.43.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,107 @@
 Release History
 ===============
 
+2.43.0
+++++++
+
+**AKS**
+
+* `aks enable-addons`: Add `--enable-syslog` parameter to monitoring addon (#24320)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az aks enable-addons`: Add `--enable-syslog` parameter to monitoring addon (#24320)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,107 @@
...
+
+* `az monitor diagnostic-settings`: Migrate show, delete, list cmds (#24725)
+
+**Network**
+
+* `az network bastion rdp`: Allow rdp session customization (#24434)
+* `az network private-endpoint-connection`: Enable private link support for provider `Microsoft.DesktopVirtualization/hostpools` and `Microsoft.DesktopVirtualization/workspaces` (#24568)
+* `az network application-gateway`: Support OCSP revocation check on client certificate (#24556)
+* `az network traffic-manager endpoint`: Add `--always-serve` to manage the health check on endpoints (#24716)
+* `az network public-ip create`: Fix the issue when using `--ip-tags` (#24728)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:74]: Please specify what the issue is.

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,107 @@
...
+* `az iot hub create/delete`: Add `--no-wait` parameter to support no wait operation (#24261)
+
+**Key Vault**
+
+* `az keyvault`: Add check-name command, support Security Domain Properties (#24636)
+
+**Misc.**
+
+* Change name of Bash completion function (#23287)
+
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:63]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,107 @@
...
+**Monitor**
+
+* `az monitor diagnostic-settings`: Migrate show, delete, list cmds (#24725)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:62]:
```suggestion
* `az monitor diagnostic-settings`: Add `--marketplace-partner-id` parameter (#24725)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,107 @@
...
+* `az network public-ip create`: Fix the issue when using `--ip-tags` (#24728)
+* `az network private-endpoint-connection`: Add Provider `Microsoft.MachineLearningServices/registries` (#24712)
+
+**RDBMS**
+
+* `az postgres flexible-server geo-restore/replica`: Introduce read replicas and geo-restore (#24639)
+* `az postgres flexible-server upgrade`: Add major version upgrade for PostgreSQL flexible server (#24649)
+* `az postgres flexible-server create/update/restore/replica`: Postgres flex byok (#24651)
+* `az postgres flexible-server identity`: User managed identity operations for PostgreSQL flexible server (#24713)
+* `az postgres flexible-server create/update/ad-admin`: Azure Active Directory Administrator operations for PostgreSQL flexible server (#24713)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:79]:
```suggestion
* `az postgres flexible-server identity`: Add user managed identity operations for PostgreSQL flexible server (#24713)
* `az postgres flexible-server create/update/ad-admin`: Add Azure Active Directory Administrator operations for PostgreSQL flexible server (#24713)
```

**Comment by yonzhan** [comment]
Azure CLI 2.43.0 release

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,99 @@
...
+
+* `az monitor diagnostic-settings`: Migrate show, delete, list cmds (#24725)
+
+**Network**
+
+* `az network bastion rdp`: Allow rdp session customization (#24434)
+* `az network private-endpoint-connection`: Enable private link support for provider `Microsoft.DesktopVirtualization/hostpools` and `Microsoft.DesktopVirtualization/workspaces` (#24568)
+* `az network application-gateway`: Support OCSP revocation check on client certificate (#24556)
+* `az network traffic-manager endpoint`: Add `--always-serve` to manage the health check on endpoints (#24716)
+* `az network public-ip create`: Fix the issue when using `--ip-tags` (#24728)
```
Comment (necusjz) [inline src/azure-cli/HISTORY.rst:66]:
```suggestion
* `az network public-ip create`: Fix `--ip-tags` cannot be used (#24728)
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**